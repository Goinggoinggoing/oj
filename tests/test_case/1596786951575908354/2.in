梅西,7,姆巴佩,10,内马尔,9,C罗,5