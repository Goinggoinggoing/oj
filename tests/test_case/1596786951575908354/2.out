请输入运动员信息：
各球员与进球数为：
梅西 7
姆巴佩 10
内马尔 9
C罗 5
球员姆巴佩进球最多为：10
